hfst-language-module/Makefile.am

```diff
@@ -186,7 +186,6 @@ pkgconfig_DATA = $(BASENAME).pc
 noinst_DATA=modes/$(LANG1)-morph.mode
 
 install-data-local: install-modes
-	test -d $(DESTDIR)$(apertium_{{languageCode}}_srcdir) || mkdir -p $(DESTDIR)$(apertium_{{languageCode}}_srcdir)
```

Comment:
Does `$(INSTALL_DATA)` do this?

Reply:
I don't actually know what does it, I just know that it's not necessary to explicitly create the target folder. And even if it was necessary, one should use `$(MKDIR_P)` instead of `mkdir` or `mkdir -p`